### ISSUE TITLE
Reset-PSFConfig returns error when called

### ISSUE THREAD

**Issue by nvarscar** [PowershellFrameworkCollective/psframework#223]
```
WARNING: [00:11:03][Reset-PSFConfig] Failed to reset the configuration item. | Method invocation failed because
[PSFramework.Configuration.Config] does not contain a method named 'Reset'.
```
It tries to call a method .Reset() internally, yet `Get-PSFConfig -Module mymodule -Name myname1 | gm` returns a method called .ResetValue()

**Comment by FriedrichWeinmann** [comment]
Heya, thanks for pointing out this bug!
Will be fixed in the next version

**Comment by FriedrichWeinmann** [comment]
Resolved in 0.10.28.144
